index.js

```diff
@@ -397,39 +397,33 @@
   //. const $ = require('sanctuary-def');
   //. const type = require('sanctuary-type-identifiers');
   //.
-  //. //    identityTypeIdent :: String
-  //. const identityTypeIdent = 'my-package/Identity';
-  //.
   //. //    Identity :: a -> Identity a
-  //. function Identity(x) {
+  //. const Identity = function Identity(x) {
   //.   if (!(this instanceof Identity)) return new Identity(x);
```

Comment:
Why the assignment here? Is it simply to emphasize that `Identity` is an object before adding a property to it? Or is it to avoid redefinition later?